### ISSUE TITLE
Quests

### ISSUE THREAD

**Issue by AefRal** [TeamAOF/All-of-Fabric-4#56]
CurseForge launcher has this pack listed with the Quests Icon, but it has no quests. Didn't read anywhere that it is planned or anything. Should remove the tag until quests are actually added.

**Comment by suduko666** [comment]
agreed getting into to the world hoping for quests only to find none kinda kills it for me

**Comment by greggdogg** [comment]
I also agree

**Comment by Nyfaria** [comment]
Quests are coming Soonisherino from what i understand.
